### ISSUE TITLE
:reset fails with "sql parser error: ..." 

### ISSUE THREAD

**Issue by dolzenko** [shshemi/tabiew#23]
I opened my file, performed `:S min(my_column)` command,  it worked, but getting back to original state with `:reset` is not working for me, fails with `sql parser error: Expected: identifier, found: 2024.11 at Line: 1, Column: 15` instead.

Thanks for the great project btw :heart:

**Comment by shshemi** [comment]
Hey,
Thank you for reporting. Would you kindly share the name of the file that you opened with Tabiew?

**Comment by technologicalsingularity** [comment]
Hey everybody, I'm try to: Q SELECT * FROM 2024-11-13_08-39-53.830246501 WHERE ra = '10.10.10.12'. This is jsonl file named 2024-11-13_08-39-53.830246501 (without file extension). I have error:  sql parser error: Expected: identifier, found: 2024 at Line: 1, Column: 15.  But sequence of S,F,O works fine. There is not first time where I have errors with polars-sql (in my other projects). How to force polars-sql work (may be I do not understand something).

**Comment by dolzenko** [comment]
Yes, my file name is 2024.11.11.pa

**Comment by shshemi** [comment]
>Yes, my file name is 2024.11.11.pa

The bug is fixed, and will work as expected in the next release.

**Comment by shshemi** [comment]
>, I'm try to: Q SELECT * FROM 2024-11-13_08-39-53.830246501 WHERE ra = '10.10.10.12'.

Please, try to execute your query as follow:
```sql
SELECT * FROM '2024-11-13_08-39-53.830246501' WHERE ra = '10.10.10.12'
```

**Comment by shshemi** [comment]
The problem is now fixed in version 0.7.1

**Comment by technologicalsingularity** [comment]
It works but SELECT * FROM '2024-11-13_08-39-53' WHERE ra = '10.10.10.12' not SELECT * FROM '2024-11-13_08-39-53.830246501' WHERE ra = '10.10.10.12'. Schema shows 2024-11-13_08-39-53. I think 830246501 interpreted as file extension.
